client/state/timezones/schema.js

```diff
@@ -0,0 +1,33 @@
+
+export const timezonesSchema = {
+	type: 'object',
+	additionalProperties: false,
+	properties: {
+		manual_utc_offsets: {
+			type: 'array',
+			items: {
+				type: 'object',
+				properties: {
+					label: { type: 'string' },
+					value: { type: 'string' }
+				}
+			}
+		},
+
+		timezones_by_continent: {
+			type: 'object',
+			patternProperties: {
+				'^[a-z]{2}$': {
+					type: 'array',
+					items: {
+						type: 'object',
+						properties: {
+							label: { type: 'string' },
+							value: { type: 'string' }
+						}
+					}
+				}
+			}
+		}
+	}
+};
```

Comment:
> this is the perfect place to use fromApi() to normalize the names

👍 We should pay attention to those redux implementations which are already assembling their data gotten from the REST API though.